impl/src/context.rs

```diff
@@ -128,6 +128,7 @@ pub struct CrateContext {
   pub features: Vec<String>,
   pub workspace_path_to_crate: String,
   pub workspace_member_dependents: Vec<PathBuf>,
+  pub workspace_member_dev_dependents: Vec<PathBuf>,
```

Comment:
This field is the cause of code changes outside of `rendering::bazel.rs`. I needed to be able to differentiate development and normal dependencies when rendering.